### PR TITLE
docs: update prometheus remote write document

### PR DESCRIPTION
**Pull request by waynexia**
## What's Changed in this PR

*Describe the change in this PR*

Update document about prometheus remote write's config

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.7/en/user-guide/write-data/prometheus.md [hunk 1]
```diff
@@ -22,11 +22,24 @@ remote_read:
...
-The `db` parameter in the url represents the database that we want to write data.
+The `db` parameter in the url represents the database that we want to write data. It's optional.
 By default, the database is `public`.
 If you want to write to another database, you can [create a new database](../table-management.md#create-database) 
 and replace `public` with the new database name.
 
+The `physical_table` parameter in the url represents the physical table that we want to write data. In GreptimeDB's `Metric` engine, we have a concept of physical table which specifies the table that store data physically. This parameter is optional. If you don't specify it, the default physical table name `greptime_physical_table` will be used. This physical table will be created automatically if it doesn't exist.
+
+Notice that this `physical_table` parameter only works when
+- `with_metric_engine` is enabled in the configuration file. It's enabled by default.
```
Comment (nicecui) [inline docs/v0.7/en/user-guide/write-data/prometheus.md:33]: Can you add the `with_metric_engine` to `configuration.md`?

File: docs/v0.7/en/user-guide/write-data/prometheus.md [hunk 2]
```diff
@@ -22,11 +22,24 @@ remote_read:
...
+- There is no previous data in corresponding `db`. I.e., those metrics to be written are first seen to that `db`.
```
Comment (nicecui) [inline docs/v0.7/en/user-guide/write-data/prometheus.md:34]: confused about what does this mean...what is `previous data`? what is `first seen`?

File: docs/v0.7/en/user-guide/write-data/prometheus.md [hunk 3]
```diff
@@ -22,11 +22,24 @@ remote_read:
 Be sure to uncomment `basic_auth` section and replace `greptime_user(username)`, `greptime_pwd(password)` with your own username and password when you enable database authentication. Please refer to client [authentication](../clients/authentication.md).
 :::
 
-The `db` parameter in the url represents the database that we want to write data.
+The `db` parameter in the url represents the database that we want to write data. It's optional.
 By default, the database is `public`.
 If you want to write to another database, you can [create a new database](../table-management.md#create-database) 
 and replace `public` with the new database name.
 
+The `physical_table` parameter in the url represents the physical table that we want to write data. In GreptimeDB's `Metric` engine, we have a concept of physical table which specifies the table that store data physically. This parameter is optional. If you don't specify it, the default physical table name `greptime_physical_table` will be used. This physical table will be created automatically if it doesn't exist.
```
Comment (nicecui) [inline docs/v0.7/en/user-guide/write-data/prometheus.md:30]: can you add the description about how to create a physical table to `table-management.md`?

**Review by nicecui — COMMENTED** [review]
Can you modify the [Time-series table](https://docs.greptime.com/user-guide/concepts/key-concepts#time-series-table) in key-concepts.md to introduce the differences between the physical tables and normal tables? also introduce their features respectively.

**Comment by waynexia** [comment]
> Can you modify the [Time-series table](https://docs.greptime.com/user-guide/concepts/key-concepts#time-series-table) in `key-concepts.md` to introduce the differences between the physical tables and normal tables? also introduce their features respectively.

I'm writing another document for metric engine which should include those concepts. I'll add a link to that section once done.

File: docs/v0.7/en/user-guide/write-data/prometheus.md (hunk 2 above)
Comment (waynexia) [reply docs/v0.7/en/user-guide/write-data/prometheus.md:34]: I wanna express that "the table didn't exist before", where the "table" references the metric to be ingested. Could you please help me to word it?

File: docs/v0.7/zh/user-guide/write-data/prometheus.md [hunk 4]
```diff
@@ -22,9 +22,22 @@ remote_read:
...
 
-url 中的 `db` 参数表示我们要写入的数据库，默认为 `public`。
+url 中的 `db` 参数表示我们要写入的数据库，是可选的，默认为 `public`。
 如果你想要写入到其他数据库，可以[创建新数据库](../table-management.md#create-database)并将 `public` 替换为新的数据库名称。
 
+url 中的 `physical_table` 参数表示我们要写入的物理表。在 GreptimeDB 的 `Metric` 引擎中，我们有一个物理表的概念，它指定了存储数据的物理表。这个参数是可选的。如果你不指定它，将会使用默认的物理表名 `greptime_physical_table`。不存在的物理表将会被自动创建。
+
+请注意，这个 `physical_table` 参数只在下面两种情况下生效：
+- 在配置文件中启用了 `with_metric_engine`。它默认是启用的。
+- 对应的 `db` 中没有之前的数据。也就是说，要写入的指标是第一次出现在这个 `db` 中。
```
Comment (nicecui) [inline docs/v0.7/zh/user-guide/write-data/prometheus.md:32]:
```suggestion
- 对应的 `db` 中没有已经存在的同名逻辑表。
```

File: docs/v0.7/en/user-guide/write-data/prometheus.md [hunk 5]
```diff
@@ -22,11 +22,24 @@ remote_read:
 Be sure to uncomment `basic_auth` section and replace `greptime_user(username)`, `greptime_pwd(password)` with your own username and password when you enable database authentication. Please refer to client [authentication](../clients/authentication.md).
 :::
 
-The `db` parameter in the url represents the database that we want to write data.
+The `db` parameter in the url represents the database that we want to write data. It's optional.
```
Comment (nicecui) [inline docs/v0.7/en/user-guide/write-data/prometheus.md:25]:
```suggestion
The `db` parameter in the URL represents the database that we want to write data. It's optional.
```

File: docs/v0.7/en/user-guide/write-data/prometheus.md (hunk 3 above)
Comment (nicecui) [inline docs/v0.7/en/user-guide/write-data/prometheus.md:30]:
```suggestion
GreptimeDB automatically groups multiple Prometheus metrics (../clients/prometheus#data-model) into the corresponding logical tables, so you do not need to specify the logical table in the URL of `remote_write`.

The `physical_table` parameter in the URL is optional and represents the [physical table] (/contributor-guide/datanode/metric-engine#physical table) where the data is written. The `greptime_physical_table` table is used by default if this parameter is not specified. If the specified physical table does not exist, the table will be automatically created.
```

File: docs/v0.7/en/user-guide/write-data/prometheus.md (hunk 2 above)
Comment (nicecui) [inline docs/v0.7/en/user-guide/write-data/prometheus.md:34]:
```suggestion
There is no existing logical table with the same name in the corresponding `db`.
```

**Review by nicecui — COMMENTED** [review]
The content that needs to be added:
* Add the `with_metric_engine` config to `configuration.md`.

The questions that needs to be discussed:
* There is no usage scene that requires creating a physical table in advance, so maybe users do not need to create a physical table manually, and users do not need to know how to create it.
* Why do we allow a logical table and a physical table can use the same table name?
* How to move from mito engine to metric engine? do users need to concern about this question when writing a physical table name in the `remote_write` URL?
* Do we support querying phsical tables? In what scenarios do we recommend users to query phycical tables?

**Comment by waynexia** [comment]
>How to move from mito engine to metric engine? do users need to concern about this question when writing a physical table name in the remote_write URL?
>Do we support querying phsical tables? In what scenarios do we recommend users to query phycical tables?

I'm not sure where to put that content. "Write data" chapter seems not suit.

**Comment by waynexia** [comment]
>Why do we allow a logical table and a physical table can use the same table name?

It's not allowed. Do I write something wrong?

**Review by nicecui — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]

**Comment by nicecui** [comment]
Due to the automated creation of the physical table, users do not need to concern about migration from mito engine to metric engine. When the names of a mito table and the physical table are the same, an error will be thrown.
`SELECT` isn't supported in physical table now, we will add this feature in the future.

**Pull request merged**